test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,75 @@
+require "test_helper"
+
+describe UsersController do
+  describe "index" do
+    it "can get the index page" do
+      get "/users"
```

Comment:
Use the path variables

```suggestion
      get users_path
```